### ISSUE TITLE
Question: Benchmark doubt

### ISSUE THREAD

**Issue by krishnaTORQUE** [bellard/quickjs#254]
I have a small doubt.

https://bellard.org/quickjs/bench.html

So according to this link.
V8 engine is much more performant?

**Comment by chqrlie** [comment]
@krishnaTORQUE: yes, I'm afraid v8 is more efficient, especially when optimizing to assembly (without `--jitless`).

QuickJS is **much** smaller, uses less memory and does not require an large c++ runtime...
We are working on several performance improvement ideas, stay tuned.
